### DISCUSSION TITLE
Mass Tag Replacer?

### DISCUSSION THREAD

**Discussion by OV-103** [AprilSylph/XKit-Rewritten#156] [Suggestion Box]
hi there, i am new to this extension because new xkit has been seemingly totally busted on firefox for months now, and so far i like it much better than anything before! i am also new to github so please correct me if i am suggesting an addition in the wrong place.
i'd like to suggest mass tag replacer as another extension, since that is the one i used the most by far. thank you for all your hard work so far! have a good one!

**Comment by AprilSylph** [comment]
[It's planned already](https://github.com/AprilSylph/XKit-Rewritten/projects/4). Please keep in mind the core framework is not finished yet and that everything is being rebuilt from scratch.

**Reply by OV-103** [reply]
oh, my mistake, i'm afraid i don't know my way around this site yet! thank you, and no rush, you're doing awesome :)
